tests/sockets/ws/README.md

```diff
@@ -93,7 +93,7 @@ wss.on('connection', function connection(ws) {
   var location = url.parse(ws.upgradeReq.url, true);
```

Comment:
`tests/sockets/ws/...` is third part code.. we should probably move it to `tests/third_party`

Reply:
Fixed